packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -295,6 +310,18 @@ function InstalledFonts() {
 							</Text>
 							<Spacer margin={ 4 } />
 							<VStack spacing={ 0 }>
+								<Flex
+									className="font-library-modal__select-all"
+									justify="flex-start"
+									align="center"
+									gap="1rem"
+								>
```

Comment:
We should be able to place the checkbox properly without using the `Flex` component by simply applying a CSS class to the `CheckboxControl` component itself.